atlasdb-config/src/main/java/com/palantir/atlasdb/factory/ServiceDiscoveringAtlasSupplier.java

```diff
@@ -83,6 +86,12 @@ public synchronized TimestampService getTimestampService() {
         return timestampService.get();
     }
 
+    public synchronized TimestampStoreInvalidator getTimestampStoreInvalidator() {
+        log.info("[timestamp-store-invalidator] Getting timestamp store invalidator from thread {}.",
```

Comment:
Yep, that's fair. Although `"This should only happen once"` is true for the invalidators, there are no serious consequences to having multiple invalidators. I'll remove it.